bot.go

```diff
@@ -85,7 +85,7 @@ func BotEndpoint(w http.ResponseWriter, r *http.Request) {
 		}
 		infoL(r, fmt.Sprintf("STAMP_CREATED(SID:%s) was received", req.ID))
 
-		if err := sendMessage(systemMessageChannelID, fmt.Sprintf("%s がスタンプ `:%s:` :%s: を作成しました", createUserMention(req.Creator), req.Name, req.Name)); err != nil {
+		if err := sendMessage(systemMessageChannelID, fmt.Sprintf("%s がスタンプ `:%s:` :%s.ex-large: を作成しました", createUserMention(req.Creator), req.Name, req.Name)); err != nil {
```

Comment:
```suggestion
		if err := sendMessage(systemMessageChannelID, fmt.Sprintf(":%s.ex-large:\n\n%s が `:%s:` を作成しました", req.Name, createUserMention(req.Creator), req.Name)); err != nil {
```

takeno_hitoさんの実装方式です
自分はこの方式で行きたいと思っていますがどうでしょうか

Reply:
@Kaki256

Reply:
```suggestion
		if err := sendMessage(systemMessageChannelID, fmt.Sprintf("%s が`:%s:`を作成しました。\n :%s.ex-large:", createUserMention(req.Creator), req.Name, req.Name)); err != nil {
```

https://q.trap.jp/messages/d8b6fa43-6234-4e81-89c9-bcc09cd45bcc

この投稿に示したバージョンです
どうでしょうか

Reply:
@Kaki256

Reply:
恐らく takeno_hito さんのやつをほとんどそのまま貼ってくれてる気がするんですが、スタンプであることは明示したほうが良いかも + 半角と全角の間にスペースを入れたほうが良いかも です (何かと干渉するわけでもないですが、一応元々の投稿文を変えるほどの意味はないかな と考えています)

```suggestion
		if err := sendMessage(systemMessageChannelID, fmt.Sprintf("%s がスタンプ `:%s:` を作成しました\n:%s.ex-large:", createUserMention(req.Creator), req.Name, req.Name)); err != nil {
```

Reply:
@alter334

Reply:
ありがとうございます
そのようにして大丈夫です お願いします

Reply:
@Kaki256